docs/developer-docs/latest/plugins/upload.md

```diff
@@ -105,11 +142,13 @@ module.exports = ({ env }) => ({
 
 </div>
 
-## Upload files
+## Examples
```

Comment:
I think I would review that whole "Examples" section to be honest. I don't think the content is just examples, all those configuration options for files imported via the Media Library should be integrated/explained properly in the Upload plugin documentation. Happy to discuss that with you!

Reply:
Same. Happy to join the discussion if you need me 🤗